src/styles/blocks/stars.css

```
@@ -0,0 +1,32 @@
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 > .stars__star:nth-child(-n + 1) {
```

```suggestion
.stars--1 > .stars__star:nth-child(-n + 1),
.stars--2 > .stars__star:nth-child(-n + 2),
.... {
```
Consider using commas to group modifiers